jest/defs.bzl

```diff
@@ -237,7 +237,7 @@ def jest_test(
             key: val
             for key, val in kwargs.items()
             # List from https://bazel.build/reference/be/common-definitions#common-attributes-tests
```

Comment:
`env` is in this list though?

Reply:
Maybe add a comment explaining why this list is different then the link?

Reply:
yes, this is a filter out list. env should be passed through. will add comment